src/platform/haswell/haswell.x.in

```diff
@@ -15,13 +15,13 @@ OUTPUT_ARCH(xtensa)
 MEMORY
 {
   vector_reset_text :
-        org = XCHAL_RESET_VECTOR0_PADDR,
+        org = XCHAL_RESET_VECTOR_PADDR,
```

Comment:
What is the reason of this change?

Reply:
change in core-isa.h, VECTOR0 no longer exists.